README.md

```diff
@@ -41,7 +41,7 @@ Once vgreped, you can perform certain operations on the results such as limiting
 Enter a vgrep command: ?
 vgrep command help: command[context lines] [selectors]
          selectors: '3' (single), '1,2,6' (multi), '1-8' (range)
-          commands: print, show, context, tree, delete, keep, refine, files, quit, ?
+          commands: print, show, context, tree, delete, keep, refine, files, grep, quit, ?
```

Comment:
I think this should be `newsearch` but I actually prefer `grep` over `newsearch`.

Reply:
Oops! So I went with `grep` first and thought it might be confused with `refine` (as in “grep from result list”) and switched to `newsearch`... Except I forgot this one.

Ok, will go back to `grep` in that case.